src/index.ts

```diff
@@ -1 +1,60 @@
-// hello world
+/**
+ * Italia PagoPA Proxy
+ * Cittadinanza Digitale PagoPA services
+ *
+ *
+ */
+import * as debug from "debug";
+import * as http from "http";
+import { newApp } from "./App";
+import { CONFIG } from "./configuration";
+
+debug("ts-express:server");
+
+const port = normalizePort(process.env.PORT || CONFIG.CONTROLLER.PORT);
+const app = newApp();
+
+app.set("port", port);
+
+const server = http.createServer(app);
+server.listen(port);
+server.on("error", onError);
+server.on("listening", onListening);
+
+function normalizePort(val: number | string): number | string | boolean {
```

Comment:
you probably want to return an `Option<number>` or an `Either<Error, number>` here